library/src/handle.cpp

```diff
@@ -18,12 +19,13 @@
 _rocblas_handle::_rocblas_handle()
 {
 #if BUILD_WITH_TENSILE
-    static int dummy = (tensileInitialize(), 0);
```

Comment:
`tensileInitialize()` still needs to be called in new client, right?

Reply:
I will move `tensileInitialize()` to `tensile_host.cpp` so that all references to Tensile are in `tensile_host.cpp` in the new Tensile client.